debezium-core/src/test/java/io/debezium/data/VerifyRecord.java

```diff
@@ -1090,7 +1090,7 @@ private static boolean areConnectSchemasEqual(Schema schema1, Schema schema2) {
             valueSchemasEqual = Objects.equals(schema1.valueSchema(), schema2.valueSchema());
         }
         else if (schema1.type() == Type.ARRAY && schema2.type() == Type.ARRAY) {
-            valueSchemasEqual = Objects.equals(schema1.valueSchema(), schema2.valueSchema());
+            valueSchemasEqual = areConnectSchemasEqual(schema1.valueSchema(), schema2.valueSchema());
```

Comment:
I've pushed a commit for changing the previous string-based comparison to calling this method again for the value schema. It revealed one glitch in the expected schema for the UPDATE case. Could you double-check that last commit I've added? Thanks!

Reply:
Ah, I don't know why I didn't think to just recurse on this method. This seems like the way to go. Thanks!